samples/rats-tls-client/client.c

```diff
@@ -54,7 +54,8 @@ static sgx_enclave_id_t load_enclave(bool debug_enclave)
 
 int rats_tls_client_startup(rats_tls_log_level_t log_level, char *attester_type,
 			    char *verifier_type, char *tls_type, char *crypto_type, bool mutual,
-			    bool debug_enclave, char *ip, int port, bool verdictd)
+			    bool provide_endorsements, bool debug_enclave, char *ip, int port,
```

Comment:
Change `provide_endorsements` to `endorsements` seems look good.

Reply:
This has been resolved by changing `--provide-endorsements` to `--endorsements`